SwiftLeeds/Network/SwiftLeedsEnvironment.swift

```diff
@@ -9,5 +9,11 @@ import Foundation
 import NetworkKit
 
 struct SwiftLeedsEnvironment: NetworkEnvironmentProviding {
-    let apiUrl: String = "https://rickandmortyapi.com/api"
+    let apiUrl: String = "https://www.swiftleeds.co.uk/api/v1"
```

Comment:
This should be apiURL ( I think I added this line, but still good to sort )

Reply:
It's calling the API, so no idea why it would be appUrl, prefer this myself

Reply:
Read it again 😛

Reply:
Ah got you, my eyes are fading.  Will update that 👍

Reply:
Ah, got you.  I'll have to start using dynamic type 😆